packages/mdl-snackbar/mdl-snackbar.scss

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import "mdl-animation/mixins";
+@import "mdl-theme/mixins";
+@import "mdl-typography/mixins";
+@import "./variables";
+
+/* postcss-bem-linter: define snackbar */
+.mdl-snackbar {
```

Comment:
Some general comments on the styling:

- The default snackbar actually has a width of `100%` and no `left` parameter set. This is the way it's supposed to look on smaller screens. 
- The snackbar height shouldn't be variable. Either it's a single-line snackbar @ `48px`, or a multi-line snackbar @ `80px`. When I've written snackbar elements in the passed, I've used a `--multiline` modifier class to accomplish this. This means that in the foundation class you could have a `setMultiline()` method, and a `multiline` setter in the component which could delegate to the foundation method.
- It's probably a good idea to set `overflow:hidden` on the snackbar so that if people start putting more than two lines, the element doesn't become scrollable.